docs/data/sources/google-tag-manager-server.md

```diff
@@ -8,10 +8,144 @@ This is the server-side Google Tag Manager Template for Amplitude Analytics. The
 !!!info Resources
     [:simple-googletagmanager: GTM Template Gallery](https://tagmanager.google.com/gallery/#/owners/amplitude/templates/amplitude-server-gtm-template) · [:material-github: GitHub](https://github.com/amplitude/amplitude-server-gtm-template)
 
-This video tutorial walks through the implementation basics. 
+!!!note
+    Ensure to consistently update your Amplitude GTM template to the latest version for an enhanced feature set, crucial bug fixes, and a significantly improved user experience.
+
+## Workflow
+
+### Container Setup
+
+If you start from zero, you need to setup your conatiner first. This Amplitude Analytics tag template can be found in **Server** target platform which for the uses on desktop and mobile web pages.
+
+![Server Container Setup](../../assets/images/gtm/gtm-server-conatiner-setup.png)
+
+### Add Template
+
+Create a new tag template by searching the gallery. Choose Amplitude Analytics and click `add` button.
+![Amplitude Analytics](../../assets/images/gtm/gtm-amplitude-analytics.png)
+
+### Create a Client
```

Comment:
Feel like [this](https://developers.google.com/tag-platform/learn/sst-fundamentals/2-what-is-sst) is really helpful to understand how server-side GTM works. And why clients are needed. Maybe have this link here?